ci/build-docs.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-tracked_markdown=$(git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$')
+tracked_markdown=$(mktemp)
+git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$' > $tracked_markdown
+trap "EXIT" "rm ${tracked_markdown}"
```

Comment:
`trap` works like: `trap "thing to evaluate" list of signals`

```suggestion
trap "rm ${tracked_markdown}" "EXIT"
git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$' > $tracked_markdown
```